hs_data_services/hs_data_services_sync/management/commands/copy_res_files_from_hydroshare.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import json
+from django.core.management.base import BaseCommand
+from hs_data_services import settings
+from hs_data_services_sync import tasks
+
+
+class Command(BaseCommand):
+    help = "Copy resource files from HydroShare to geoserver storage"
+
+    def add_arguments(self, parser):
+        parser.add_argument('resource_ids', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+        resource_ids = options['resource_ids']
+        if len(resource_ids) > 0:
+            for resource_id in resource_ids:
+                print(f"Copying files from resource: {resource_id}")
+                result = self.copy_files(resource_id)
+                print(result)
+        else:
+            resources = self.get_list_of_public_geo_resources()
+            num_resources = len(resources)
+            print(f"Copying files from {num_resources} resources")
+            counter = 1
+            for res_id in resources:
+                print(f"{counter}/{num_resources} - Copying files from resource: {res_id}")
+                result = self.copy_files(res_id)
+                print(result)
+                counter += 1
+        print("Done copying files")
+
+    def get_list_of_public_geo_resources(self):
+        hydroshare_url = "/".join(settings.HYDROSHARE_URL.split("/")[:-1])
+        types = ["Geographic Feature (ESRI Shapefiles)", "Geographic Raster"]
+        # replace spaces with + for the query string
+        types = [t.replace(" ", "+") for t in types]
+        params = {
+            "type": types,
+            "availability": ["public"],
+            "geofilter": "false"
+        }
+        rest_url = f"{hydroshare_url}/discoverapi/?filter={json.dumps(params)}"
+        rest_url = rest_url.replace(" ", "")
+        print(f"Getting list of public geospatial resources from: {rest_url}")
```

Comment:
the parsing of parameters for our discoveryapi is really odd. I was using `urlib` initially but couldn't get it to work so just did this manually.

I couldn't use `/hsapi/resources` because the query param for `content_type` seems broken...unless I just couldn't figure it out in Swagger. Maybe a good use here for hsclient, but since the other hs calls in this repo are straight requests to hs api, I decided to stick with it